institutions-api/src/main/scala/hmda/institution/projection/InstitutionDBProjection.scala

```diff
@@ -30,6 +31,8 @@ object InstitutionDBProjection
   implicit val institutionEmailsRepository = new InstitutionEmailsRepository(
     dbConfig)
 
+  implicit val ec: ExecutionContext = ExecutionContext.global
```

Comment:
This could potentially produce thread starvation under load, but until we have measurements it's hard to tell. I wouldn't change it for now, but let's try to keep in mind that if we see performance issues or errors with this approach it might be because of that issue.